app/views/user_mailer/new_auth0_user_new_project.html.erb

```diff
@@ -3,6 +3,9 @@
 <p><%= @sharing_user_name %> (<%= @sharing_user_email %>) has invited you to IDseq project
   <b><a href="https://idseq.net/my_data?projectId=<%=(@shared_project_id)%>"><%= @shared_project_name %></a></b>.</p>
 <p></p>
-<p><b><%= link_to 'Activate your account', @reset_password_url %></b></p>
+<p><b><%= link_to 'Activate Your Account', @reset_password_url %></b></p>
```

Comment:
Updated this one to be consistent with the design choices we agreed on, like capitalizing "Activate Your Account", changing wording.